src/test/java/HomeWork/pages/DynamicControlsPage.java

```diff
@@ -38,7 +41,20 @@ public void clickInputButton() {
         driver.findElement(INPUT_BUTTON).click();
     }
 
-    public boolean isInputDisabled(WebDriver driver, By by) {
-        return driver.findElement(by).getAttribute("disabled") != null;
+    public boolean isInputAvailable(By by) {
```

Comment:
So in this method you don't need an input arguments at all, you can use the locator from this class inside the method:
return driver.findElement(INPUT_LOCATOR).isEnabled()